go.mod

```diff
@@ -5,11 +5,12 @@ go 1.14
 require (
 	github.com/evanphx/json-patch v4.5.0+incompatible
 	github.com/ghodss/yaml v1.0.0
-	github.com/giantswarm/apiextensions v0.4.9
+	github.com/giantswarm/apiextensions v0.4.12-0.20200709143104-8685f018ea1f
```

Comment:
TODO: Change to new release once https://github.com/giantswarm/apiextensions/pull/513 is merged and a new release is available

Reply:
I saw you released v0.4.13, I updated it in the PR